inference-engine/src/gna_plugin/layers/gna_layer_info.hpp

```diff
@@ -145,7 +145,7 @@ class LayerInfo {
         return false;
     }
     bool isConst() const noexcept {
-        return isOfType("const");
+        return isOfType("const") || isOfType("Const");
```

Comment:
Probably this change is not needed. Need to double check.

Reply:
Yes, you're right - isOfType uses caseless comparison. I'll remove it.